magicctapipe/io/io.py

```diff
@@ -249,6 +251,53 @@ def format_object(input_object):
     return string
 
 
+def get_custom_cuts(config, energy_bins):
+    """
+    Extract and interpolate custom, energy-dependent cuts for the IRFs creation.
+
+    Parameters
+    ----------
+    config : dict
+        Dictionary containing the custom cuts and the interpolation tragedy.
```

Comment:
while nearest point interpolation is not very precise I would not call it "tragedy" ;-), I guess you meant "strategy"